Controller/AdminResettingController.php

```diff
@@ -27,108 +31,127 @@ class AdminResettingController extends ResettingController
      */
     public function requestAction()
     {
-        if ($this->container->get('security.context')->isGranted('IS_AUTHENTICATED_FULLY')) {
-            return new RedirectResponse($this->container->get('router')->generate('sonata_admin_dashboard'));
+        if ($this->isGranted('IS_AUTHENTICATED_FULLY')) {
+            return new RedirectResponse($this->generateUrl('sonata_admin_dashboard'));
         }
 
-        return $this->container->get('templating')->renderResponse('SonataUserBundle:Admin:Security/Resetting/request.html.'.$this->getEngine(), array(
-            'base_template' => $this->container->get('sonata.admin.pool')->getTemplate('layout'),
-            'admin_pool'    => $this->container->get('sonata.admin.pool'),
+        return $this->render('SonataUserBundle:Admin:Security/Resetting/request.html.twig', array(
+            'base_template' => $this->get('sonata.admin.pool')->getTemplate('layout'),
+            'admin_pool'    => $this->get('sonata.admin.pool'),
         ));
     }
 
     /**
      * {@inheritdoc}
      */
-    public function sendEmailAction()
+    public function sendEmailAction(Request $request)
```

Comment:
Parent class ```FOS\UserBundle\Controller\ResettingController``` from FOSUser2 has this [signature](https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/Controller/ResettingController.php#L43): ```public function sendEmailAction(Request $request)```.

Without this fix we get the following error:
> Runtime Notice: Declaration of Sonata\UserBundle\Controller\AdminResettingController::sendEmailAction() should be compatible with FOS\UserBundle\Controller\ResettingController::sendEmailAction(Symfony\Component\HttpFoundation\Request $request)

Please note that this PR is for branch add_support_for_fos_user2 which is entirely a BC break.